Documentation/Books/Manual/Installation/Windows.md

```diff
@@ -138,31 +150,37 @@ the *arangod.exe* executable.
 *&lt;ROOTDIR&gt;\etc\arangodb\*. Please adjust this to your needs if you want to
 use different connection settings etc.
 
-Uninstalling
-------------
+### Uninstalling
 
 To uninstall the Arango server application you can use the windows control panel
 (as you would normally uninstall an application). Note however, that any data
 files created by the Arango server will remain as well as the *&lt;ROOTDIR&gt;*
 directory.  To complete the uninstallation process, remove the data files and
 the *&lt;ROOTDIR&gt;* directory manually.
 
-Limitations for Cygwin
-----------------------
-
-Please note some important limitations when running ArangoDB under Cygwin:
-Starting ArangoDB can be started from out of a Cygwin terminal, but pressing
-*CTRL-C* will forcefully kill the server process without giving it a chance to
-handle the kill signal. In this case, a regular server shutdown is not possible,
-which may leave a file *LOCK* around in the server's data directory.  This file
-needs to be removed manually to make ArangoDB start again.  Additionally, as
-ArangoDB does not have a chance to handle the kill signal, the server cannot
-forcefully flush any data to disk on shutdown, leading to potential data loss.
-When starting ArangoDB from a Cygwin terminal it might also happen that no
-errors are printed in the terminal output.  Starting ArangoDB from an MS-DOS
-command prompt does not impose these limitations and is thus the preferred
-method.
-
-Please note that ArangoDB uses UTF-8 as its internal encoding and that the
-system console must support a UTF-8 codepage (65001) and font. It may be
-necessary to manually switch the console font to a font that supports UTF-8.
+Installing using the ZIP archive (XCopy installation)
+-----------------------------------------------------
+
+Not all users prefer the guided _Installer_ to install ArangoDB. In order to have a
+[portable application](http://en.wikipedia.org/wiki/Portable_application), or easily
+start different ArangoDB versions on the same machine, and/or for the maximum flexibility,
+you might want to install using the _ZIP_ archive ([XCOPY deployment](http://en.wikipedia.org/wiki/XCOPY_deployment)).
+
+### Unzip the archive
+
+Open an explorer, choose a place where you would like ArangoDB to be, and extract the
+archive there. It will create its own toplevel directory with the version number in the string.
+
+### Edit the configuration
+
+This step is optional. If the default configuration of ArangoDB do not suite your needs,
+you might edit `etc\arangodb3\arangod.conf` to change some configuration options.
+
+### Start the Server
+
+After installation, you may start ArangoDB in several ways. The exact start-up command
+depends on the type of ArangoDB deployment you are interested on (_Single Instance_, _Master-Slave_,
+_Active Failover_ or Cluster. 
```

Comment:
```suggestion
(_Single Instance_, _Master-Slave_, _Active Failover_ or Cluster).
```
should cluster also be italic, as in _Cluster_? Do we consider it as term coined by us?

Reply:
yes, I did not include DC2DC here because it is not supported for win